distro/commands/wiring.go

```diff
@@ -55,8 +55,14 @@ See https://projectriff.io and https://github.com/knative/docs`,
 		ImagePull(&imageClient),
 	)
 
+	system := System()
```

Comment:
Wondering if "system" is the best word to use. Although this is the `distro` tool, the word "system" here would have a different meaning than in `riff`.

Why not make `download` a level 1 command? This way, users run `riff-distro download`?